tripal/src/TripalImporter/Attribute/TripalImporter.php

```diff
@@ -70,6 +70,8 @@ class TripalImporter extends Plugin {
    *   necessary to require that the user provide a file for uploading if the
    *   loader keeps track of previous files and makes those available for
    *   selection.
+   * @param array|null $publish
+   *   An array containing a list of content types to be published after import.
```

Comment:
This needs to be updated now :-)

```suggestion
   *   An associative array indicating what Tripal Content should be published
   *   after import. It is keyed by the list type and the value indicates what
   *   to publish using that approach. List Types supported:
   *   - bundle (array): a list of bundle machine names to publish.
```